app/services/induction/amend_participant_cohort.rb

```diff
@@ -147,11 +167,9 @@ def in_target_schedule?(induction_record)
 
     def participant_declarations
       return false unless participant_profile
+      return @participant_declarations if instance_variable_defined?(:@participant_declarations)
```

Comment:
not sure I understand why this change was required?

Reply:
You mean the format change below to put it all in a single line?
I don't mind to revert it back to multiline. We have changed this method so many times back and forth during the live of this PR that it has ended up like this.

Reply:
The memoization doesn't work when the value is false